src/components/Header/Header.tsx

```diff
@@ -0,0 +1,32 @@
+import { FC } from 'react';
+
+type Props = {
+  handleSubmit: (event: React.FormEvent) => void,
+  query: string,
+  setQuery: (title: string) => void,
+};
+
+export const Header:FC<Props> = ({
+  handleSubmit,
+  query,
+  setQuery,
+}) => (
+  <header className="todoapp__header">
+    {/* this buttons is active only if there are some active todos */}
+    {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
+    <button
+      type="button"
+      className="todoapp__toggle-all active"
```

Comment:
The 'todoapp__toggle-all' button should have an onClick event handler to toggle all todos. Please add an onClick handler function. 🔄